protos/feast/core/CoreService.proto

```diff
@@ -163,23 +154,24 @@ message ListStoresResponse {
 }
 
 message ApplyFeatureSetRequest {
-    // Feature set version and source will be ignored
     feast.core.FeatureSet feature_set = 1;
 }
 
 message ApplyFeatureSetResponse {
     enum Status {
-        // Latest feature set version is consistent with provided feature set
+        // Latest feature set is consistent with provided feature set
         NO_CHANGE = 0;
```

Comment:
Out of scope for this PR, but should this be `UNKNOWN` with `NO_CHANGE` moved to a higher number?

Reply:
I dont want to break this enum

Reply:
Fair enough, but should this be tracked in an issue so that it can be fixed over time?

Reply:
okay

Reply:
A `TODO` would also suffice.